cmd/vdev_id/vdev_id

```diff
@@ -277,7 +298,8 @@ sas_handler() {
 		return
 	fi
 
-	CHAN=`map_channel $PCI_ID $PORT`
+	ENCLOSURE=`echo $end_device_dir | awk -F: '{gsub(/.*\//,""); print $2}'`
```

Comment:
This was just a re-posting of nedbass' patch, just as a pull request rather than a link to a .patch file. Frankly I'd love to rewrite it entirely. Probably in perl because that's my thing over bash, but adding a perl dependency might not be desirable